source/parser.hera

```diff
@@ -529,14 +536,23 @@ CallExpression
       type: "CallExpression",
       children: $0,
     }
-  MemberExpression CallExpressionRest* ->
-    if ($2.length) {
+
+  MemberExpression:member NonSuppressedTrailingMemberExpressions:trailing CallExpressionRest*:rest ->
+    if (rest.length) {
       return {
         type: "CallExpression",
-        children: [$1].concat($2),
+        children: [member, ...trailing, ...rest],
       }
     }
-    return $1
+
+    if (trailing.length) {
+      return {
+        children: [member, ...trailing]
```

Comment:
I think this should be a `type: "MemberExpression"`.

You could plausible combine the two cases into `if (rest.length || trailing.length)` and have `type: rest.length ? "CallExpression" : "MemberExpression"`.